lib/provisioning/Provisioner.js

```diff
@@ -565,10 +565,13 @@ Provisioner.prototype.queryLink = Promise.coroutine(function*(options) {
     return queryInfo;
 });
 
-// Get the list of currently configured networks (from the config)
+// Get the list of currently network instances
 Provisioner.prototype.queryNetworks = Promise.coroutine(function*() {
+    let thirdParty = yield this._ircBridge.getThirdPartyProtocol();
+
     return {
-        servers: this._ircBridge.ircServers.map((server) => {return server.domain})
+        bot_user_id: thirdParty.bot_user_id,
```

Comment:
Having done [theabove] you can still do this here, by

    bot_user_id: thirdParty.instances[0].bot_user_id,